.env.sample

```diff
@@ -13,3 +13,5 @@ LOG_LEVEL="debug"
 # QA Environment Variables
 IPV_STUB_URL=
 CUSTOM_FE_URL=
+TEST_HARNESS_URL=
+SESSION_TABLE="session-bav-cri-ddb"
```

Comment:
Do we want to be committing an value in the .env.sample file?

Reply:
I did because it actually seems to be the same in every env so just makes it easier but happy to remove

Reply:
Makes sense if the value stays consistent